src/App.css

```diff
@@ -1,5 +1,6 @@
 .App {
```

Comment:
Css looks nice but on the view I have the vote buttons get crossed over a little and the distance between the button and the distance between the buttons and the votes is a little large
![image](https://user-images.githubusercontent.com/22796757/199685340-1b633aa6-78b8-42db-b8db-8ce944716fc3.png)

Reply:
Thank you for pointing this out